oteapi/models/secretconfig.py

```diff
@@ -1,44 +1,12 @@
 """AttrDict for specifying user credentials or secrets."""
-import json
-from typing import TYPE_CHECKING, Optional
+from typing import Optional
 
-from pydantic import BaseModel, Field, SecretStr
+from pydantic import BaseModel, Field, SecretStr, field_serializer
 
 from oteapi.settings import settings
 
-if TYPE_CHECKING:  # pragma: no cover
-    from typing import Any, Callable
 
-
-def json_dumps(model: dict, default: "Callable[[Any], Any]") -> "str":
```

Comment:
Check if external people are using this, consider aliasing this

Reply:
As far as I know, no external people are calling or using this function directly.